source/CardGames.py

```diff
@@ -104,6 +104,12 @@ def clearHand(self):
     self.hand = []
     self.knownCards = []
 
+  def info(self):
+    print("Player's name: " + Jacob.name)
```

Comment:
Shouldn't this be print("Player's name: "  self.name)?? 
I could be wrong but I'm curious as to why you have Jacob.name.

Reply:
Yeah, I was testing it outside the code and forgot to change it to self. I believe I changed it.